core/kernel/persistence/smoothsql/class.SmoothRdf.php

```diff
@@ -165,4 +168,24 @@ protected function tripleToValue(core_kernel_classes_Triple $triple)
             'epoch' => $this->getPersistence()->getPlatForm()->getNowExpression()
         ];
     }
+
+    protected function getTripleParameterTypes()
+    {
+        return [
```

Comment:
What about move this to a constant? It is used in production code and tests?

Reply:
why not

Reply:
it is used during query to ontology so prod and maybe test